src/main/scala/io/iohk/ethereum/nodebuilder/NodeBuilder.scala

```diff
@@ -451,6 +454,15 @@ trait ProofOfStakeMinerBuilder {
     electionManager))
 }
 
+trait SlotCalculatorBuilder {
```

Comment:
I guess we need to update this name, because the builder now is building a SlotTimestampConverter instead.